pkg/api/const.go

```diff
@@ -247,11 +247,11 @@ const (
 	// DefaultWindowsPublisher sets the default WindowsPublisher value in WindowsProfile
 	DefaultWindowsPublisher = "MicrosoftWindowsServer"
 	// DefaultWindowsOffer sets the default WindowsOffer value in WindowsProfile
-	DefaultWindowsOffer = "WindowsServerSemiAnnual"
+	DefaultWindowsOffer = "WindowsServer"
```

Comment:
I don't think this should have changed. Checking with team publishing the images

Reply:
The publishing team intended to change this, so I'm updating the docs to match.